.github/workflows/post-merge-push.yaml

```diff
@@ -0,0 +1,58 @@
+name: Build latest binaries
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-push:
+    concurrency:
+      group: build-${{ github.head_ref || github.run_id }}
+      cancel-in-progress: true
+    name: Build and Unit Test
+    env:
+      REGISTRY_REPO: oci://ghcr.io/k0rdent/kcm/charts-ci
```

Comment:
NIT: it's better to explicitly put env to the step which uses it, right now it's the `Prepare and push KCM template charts to the public repository` step only